frontend/src/citizen-frontend/messages/MessagesPage.tsx

```diff
@@ -79,15 +85,21 @@ export default React.memo(function MessagesPage() {
     setSelectedThread(params.threadId)
   }, [setSelectedThread, params.threadId])
 
+  const initThreadFocusRef = useRef<() => void>()
+
```

Comment:
Eikös tämän saisi hoidettua myös vähän yksinkertaisemmin: Ensin MessagePage:ssa
```suggestion
  const threadTitleRef = useRef<HTMLDivElement>(null)
```